src/unit_tests/remoted/test_secure.c

```diff
@@ -2225,20 +2235,7 @@ void test_router_message_forward_invalid_delta_json_message(void** state)
 
     router_syscollector_handle = (ROUTER_PROVIDER_HANDLE)(1);
 
-    expect_string(__wrap_router_provider_send_fb, msg, expected_message);
-    expect_string(__wrap_router_provider_send_fb, schema, syscollector_deltas_SCHEMA);
-    will_return(__wrap_router_provider_send_fb, -1);
-
-    expect_string(__wrap__mdebug2,
-                  formatted_msg,
-                  "Unable to forward message "
-                  "'{\"agent_info\":{\"agent_id\":\"001\",\"agent_ip\":\"192.168.33.20\",\"agent_name\":\"focal\"}}' "
-                  "for agent 001");
-
-    will_return(__wrap_OSHash_Get_ex_dup, NULL);
-    expect_value(__wrap_OSHash_Get_ex_dup, self, (OSHash*)1);
-    expect_string(__wrap_OSHash_Get_ex_dup, key, data->agent_id);
-
+    // This type of message must be discarded
```

Comment:
Just a question, why do we remove the message about the discarded message?

Reply:
Due to the recently changes implemented, now that is invalid, so those messages no longer happen.